examples/solana/create-and-process-workflow.ts

```diff
@@ -21,9 +20,9 @@ const client = new StakingClient();
 const signer = TxSignerFactory.getSigner('solana');
 
 async function stakeSolana(): Promise<void> {
-  if (projectId === '' || walletAddress === '') {
+  if (walletAddress === '') {
     throw new Error(
-      'Please set the projectId and stakerAddress variables in this file',
+      'Please set the privateKey and stakerAddress variables in this file',
```

Comment:
1. can we update check to look for both priv key and walletAddress
2. can we update message to reflect both priv key and walletAddress